### DISCUSSION TITLE
3D printing

### DISCUSSION THREAD

**Discussion by mikenturner** [cardonabits/haxo-hw#20] [Q&A] [answered]
Hi
Can anyone point me to the files I need for the 3D printed parts - they are probably obvious, but Im not sure exactly what I'm looking for
Thanks
Mike

**Comment by jcard0na** [comment, marked as answer]
Sure:

https://github.com/jcard0na/haxo-hw/blob/main/models/hook_rest-Part.stl
https://github.com/jcard0na/haxo-hw/blob/main/models/neck-neck.stl
https://github.com/jcard0na/haxo-hw/blob/main/models/octave_rest-Part.stl
